common/src/main/java/com/wynntils/features/MythicBlockerFeature.java

```diff
@@ -27,17 +29,14 @@ public MythicBlockerFeature() {
 
     @SubscribeEvent
     public void onChestCloseAttempt(InventoryKeyPressEvent e) {
-        if (!WynnUtils.onWorld() || !McUtils.mc().options.keyInventory.matches(e.getKeyCode(), e.getScanCode())) return;
-        if (!(McUtils.mc().screen instanceof AbstractContainerScreen<?>)) return;
-        String title = McUtils.mc().screen.getTitle().getString();
-        if (!title.startsWith("Loot Chest")
-                && !title.startsWith("Daily Rewards")
-                && !title.contains("Objective Rewards")) return;
+        if (!WynnUtils.onWorld()) return;
+        if (!McUtils.mc().options.keyInventory.matches(e.getKeyCode(), e.getScanCode())) return;
```

Comment:
not necessarily related to this PR (which i think is good), but we should also check the menu key (esc) here

Reply:
I opened https://github.com/Wynntils/Artemis/issues/65 to keep track of this.